examples/simple-game-server/Makefile

```diff
@@ -51,26 +58,45 @@ root_path = $(realpath $(project_path)/../..)
 #                 |___/
 
 ifeq ($(WITH_WINDOWS), 1)
-push: push-multi-arch-image
+push_server_manifest += $(foreach windows_version, $(WINDOWS_VERSIONS), $(server_tag)-windows_amd64-$(windows_version))
+endif
+ifeq ($(WITH_ARM64), 1)
+push_server_manifest += $(server_tag_linux_arm64)
+endif
+
+push: push-image
```

Comment:
Is there a reason to have both `push` and `push-image` targets? Can we remove this and rename the `push-image` ones below to `push`?

Reply:
I delete redondant target